### DISCUSSION TITLE
Increasing Print Speed

### DISCUSSION THREAD

**Discussion by eecue** [Jyers/Marlin#713] [Q&A] [answered]
I've recently added a Micro Swiss direct drive kit and a lighter pancake extruder stepper as well as dual Z drives.  I'm getting great prints but I'd love to increase my print speeds.  What are the best ways to do that? Should I tweak acceleration and jerk? Do I do this in the slicer no or on the printer?

**Comment by Jyers** [comment, marked as answer]
Funny you should ask, having just added linear advance to my setup I did some speed torture tests yesterday. Here's what I found made the biggest difference:

* Acceleration - Slight improvement with diminishing returns as you increase
* Jerk - Large improvement but only at higher print speeds and at the cost of quality and printer strain
* Retractions - Large impact, get them as small and fast as will give you good results
* "Fill gaps between walls" and "Filter tiny out gaps" - Large difference depending on model
* Combing - Play around with modes to see what provides best times

Using these settings and pretty much maxed out speed I was able to print a benchy in under 50 minutes and a calibration cube in about 15.
